docs/api/connectivity/bluetooth/Optimising_for_performance.md

```diff
@@ -0,0 +1,110 @@
+
+The most intuitive power consumption rate to understand is when using connections. Each device needs to send a packet at
+below supervision timeout frequency to maintain a connection regardless of data transfer. This is the absolute minimum
+power consumption. Additionally, normally power is consumed at every connection event (exchange of packets between
+central and peripheral).
+
+It's worth considering if keeping the connection active is worth it. Connection in BLE is extremely fast and if you plan
+to only send a quick burst of data every minute it is better to connect and disconnect each time.
+
+The cost of the connection is proportionate to the negotiable connection interval. This can be set during `connect` or
+later through `updateConnectionParameters`. The higher the interval the more often radio is active. This is especially
+important for the peripheral which needs to enable the radio to receive packets.
+
+This can be further helped through setting a high `slaveLatency` parameter. This allows the peripheral to skip
+connection events and save power not just by not sending any packets but by not even listening. This is not free for
+central as it increases latency of data transmission. Central may have to attempt sending data multiple times before the
+peripheral accepts the transmission. 
+
+### Advertising and scanning
+
+Power draw during advertising is proportional to the advertising interval. Additionally, if the advertising is
+connectable or scannable it means the advertiser needs to listen on the radio after each advertisement for potential
+connection of scan requests.
+
+Scanning power draw is proportional to time spent scanning. The interaction between scanning an advertising means that
+the less power the advertiser spends advertising, the more power the scanner will have to spend to see the advertising
+packets. The decision on balance will be dictated by your design of your devices (which one is more constrained).
+
+### Connection vs advertising
+
+Instead of connecting to the device you can consider transferring data in advertising packets. This depends on the
+nature of the data.
+
+A transfer over a connection will allow you to use the ATT protocol, this can handle acknowledgement for you. This might
+be a good choice if you're sending data that must get through reliably.
+
+If your data is non-critical then advertising might be cheaper. You might have to accept less reliability and no built
+in acknowledgment. Additional benefit is that multiple devices may receive the data and each scanner may make their own
+decisions about power consumption.
+
+### Periodic advertising
+
+Periodic advertising allows you to get best of both worlds by having the power characteristics of advertising for the
+peripheral but also saving power for the scanner. After finding periodic advertising through `createSync` the scanner
+will only have to turn on the radio when the expected packet is due.
+
+## Increasing throughput
+
```

Comment:
It would be worth touching on write/write w/o response and notify/indicate as these have the possibility of dramatically increase throughput.